lib/backburner/deferred-action-queues.js

```diff
@@ -18,6 +18,10 @@ function noSuchQueue(name) {
   throw new Error('You attempted to schedule an action in a queue (' + name + ') that doesn\'t exist');
 }
 
+function noSuchMethod(name) {
+  throw new Error('You attempted to schedule an action in a queue (' + name + ') for a method that doesn\'t exist');
```

Comment:
Including the name that didn't exist would be useful in helping the user resolve it, but I'm not sure where this is called from (on my phone), so maybe that is impractical. How about something in the style of this:

> You tried to call the method `myMethod` on `target` in queue `afterRender`, but the method did not exist